src/main/java/org/wso2/carbon/inbound/poll/dbeventlistener/DBEventPollingConsumer.java

```diff
@@ -38,6 +38,8 @@
 import java.sql.Statement;
 import java.sql.DriverManager;
 import java.sql.Types;
+import java.sql.Clob;
+
```

Comment:
Shall we remove this extra line?

Reply:
fix with commit 77dba88e588a7ed6959f3ce479e96a3abb87ca38